pkg/runner/testdata/local-action-dockerfile/push.yml

```diff
@@ -1,11 +1,16 @@
 name: local-action-dockerfile
 on: push
+defaults:
+  run:
+    shell: bash
 
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - run: chmod +x ./actions/docker-local/entrypoint.sh
+    - run: chmod +x ./actions/docker-local-noargs/entrypoint.sh
```

Comment:
Couldn't this be committed as executable shell scripts and removed here?
It is not self-explaining that we do it in our test workflow.

Reply:
Maybe, but I'm on windows and face issues without these lines, act doesn't restore the exec permission for me even if I would add it.